spec/formatting.md

```diff
@@ -671,16 +672,26 @@ MAY rely on the introspection of its contents, or on other means.
 The directionality of the message as a whole is provided by the _formatting context_.
 
 If a formatted _expression_ itself contains spans with differing directionality,
-its formatter SHOULD isolate such parts to avoid
-negatively affecting the presentation of the overall formatted result.
-
-> For example, an implementation could provide a `:number` formatting function
-> which would always produce output matching the message's locale,
-> allowing for its formatted string representation to never need isolation.
+its formatter SHOULD perform any necessary processing, such as inserting controls or
+isolating such parts to ensure that the formatted value displays correctly in a plain text context.
+
+> For example, an implementation could provide a `:currency` formatting function
+> which inserts RLM, ALM, or LRM characters to properly display a currency symbol
```

Comment:
These abbreviations will not be familiar to every reader, and should be avoided esp. as this is the only place where they're used.

Reply:
Fixed. Thanks.